django_project/feti/static/js/scripts/views/map.js

```diff
@@ -126,11 +126,11 @@ define([
             this.map.on('dblclick', function (e) {
             });
 
-            L.tileLayer('https://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}', {
+            L.tileLayer('https://{s}.tile.openstreetmap.fr/osmfr/{z}/{x}/{y}.png', {
```

Comment:
hello @ann26 
is there any global openstreetmap? using com?
why we are using france domain?

Reply:
hi @meomancer , yes there is, but the france domain has prettier basemap I think. Although now this basemap: `https://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}` is working, yesterday I got 503. Maybe I should change it back?

Reply:
what do you think @cchristelis ?